apps/demo-game/src/pages/play/cockpit.tsx

```diff
@@ -161,124 +160,229 @@ function Cockpit() {
   const data_portfolio = [
     {
       cat: 'Savings',
-      val: `CHF ${resultFacts?.assets.bank.toFixed(2)}`,
+      val: `CHF ${assets.bank.toFixed(2)}`,
     },
     {
       cat: 'Bonds',
-      val: `CHF ${resultFacts?.assets.bonds.toFixed(2)}`,
+      val: `CHF ${assets.bonds.toFixed(2)}`,
     },
     {
       cat: 'Stocks',
-      val: `CHF ${resultFacts?.assets.stocks.toFixed(2)}`,
+      val: `CHF ${assets.stocks.toFixed(2)}`,
     },
     {
       cat: 'Total',
-      val: `CHF ${resultFacts?.assets.totalAssets.toFixed(2)}`,
+      val: `CHF ${assets.totalAssets.toFixed(2)}`,
     },
   ]
 
-  const header = (
-    <div className="rounded border p-4">
-      <div className="font-bold">
-        Playing as {self?.name} in game {currentGame.id}
-      </div>
+  // TODO(JJ):
+  // - Level is not consistent with component
+  // Component wants a number, level is an object, though.
+  // console.log(self)
+  // - Do onclick logic
+  // - PlayerDisplay is not ideal/nice yet
+
+  const tabs = [
+    { name: 'Welcome', href: '/play/welcome' },
+    { name: 'Cockpit', href: '/play/cockpit' },
+  ]
+
+  const playerInfo = {
+    name: data.self.name,
+    color: data.self.facts.color,
+    location: data.self.facts.location,
+    level: data.self.level.id,
+    xp: data.self.experience,
+    xpMax: data.self.experienceToNext,
+    achievements: data.self.achievements,
+    imgPathAvatar: data.self.facts.avatar,
+    imgPathLocation: `/locations/${data.self.facts.location}.svg`,
+    onClick: () => {
+      // router.replace('/play/welcome')
+    },
+  }
+
+  // Transactions
+  // TODO(JJ): Integrate into the layout
+  // The following is only temporary
+  // - Should only display final transactions, or really every decision?
+  console.log(data.result.transactions)
+  const transactions = (
```

Comment:
move this outside of this function and make it a separate CockpitTransactions component or something similar. then pass stuff into it through typed props. this would be cleaner than having it here and makes the cockpit page much shorter and easier to read